### DISCUSSION TITLE
.NET Core 7 support

### DISCUSSION THREAD

**Discussion by abcdata2010** [Finbuckle/Finbuckle.MultiTenant#627] [Q&A]
Is it support Core 7? I have an error when converting my project from Core6 to Core 7:
Error	NU1605	Detected package downgrade: Microsoft.AspNetCore.Authentication.OpenIdConnect from 7.0.0 to 6.0.12. Reference the package directly from the project to select a different version. 
 MyProject -> Finbuckle.MultiTenant.AspNetCore 6.9.1 -> Microsoft.AspNetCore.Authentication.OpenIdConnect (>= 7.0.0) 
 MyProject -> Microsoft.AspNetCore.Authentication.OpenIdConnect (>= 6.0.12)		

Thanks.
Andrey.

**Comment by abcdata2010** [comment]
It was my mistake. The conversion from core 6 to 7 was not completed in my project.  Sorry for the confusion.

**Comment by AndrewTriesToCode** [comment]
Glad you got it working!
